test/integration/custom-assertions.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+const test = require('tap').test;
+const {execCli} = require('../helper/cli');
+
+// The AssertionError constructor in Node 10 depends on the TTY interface
+test('node assertion failures are reported to the console when running in a terminal', t => {
+	const options = {
+		dirname: 'fixture/custom-assertions',
```

Comment:
Could you rename the fixtures (and this integration test) to `node-assertions`? "Custom" implies support for a wider range of assertions, like Chai.